### DISCUSSION TITLE
Can i use my own authentication methods with Web3Auth's Flutter SDK?

### DISCUSSION THREAD

**Discussion by YZhenY** [Web3Auth/Web3Auth#303] [Help]
Hello guys, 

I want to know if I can provide custom authentication using Web3Auth using Flutter SDK

**Comment by ieow** [comment]
Yes, you can!

Checkout the integration builder guide below
https://web3auth.io/docs/integration-builder?lang=flutter&chain=eth&customAuthentication=yes&whitelabel=no&dynamicConstructorParams=no&usingEmailPasswordless=no&rnWorkflowMode=expo&evmFramework=web3
